src/components/Table/TableHead.jsx

```diff
@@ -56,12 +57,15 @@ class GnoTableHead extends React.PureComponent<Props> {
               sortDirection={orderBy === column.id ? order : false}
             >
               {column.static ? (
-                column.label
+                <div style={column.style}>
+                  { column.label }
```

Comment:
we don't use spaces before/after curly braces

Reply:
done